exe-requirements.txt

```diff
@@ -11,3 +11,5 @@ pyinstaller==4.10; platform.machine == "aarch64"
 # During startup program terminated with signal SIGSEGV, Segmentation fault.
 # staying with 0.12.1 for the mean time...
 staticx==0.12.1; platform.machine == "x86_64"
+WMI==1.5.1; sys.platform == "win32"
```

Comment:
Don't need it here - we have in requirements.txt

Reply:
Same for `netifaces`

Reply:
Done